ADONET/AdoNetFundamentals/DLL/Repositories/IOrderRepository.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using AdoNetFundamentals.Entities;
+
+public interface IOrderRepository
+{
+    Order Create(Order order);
```

Comment:
Just a reminder, we can leave it as it is now: for any i/o operation (file reading, db connection, request sending and etc.) I strongly advise to use async option to prevent thread blocking till operation is processed on the other side. Just keep that in mind ;)

Reply:
That's an important advice, let me just apply this.